api/const.go

```diff
@@ -4,4 +4,8 @@ package api
 const (
 	AppName string = "app_name"
```

Comment:
@ccirello removed the C and I didn't feel like adding them back. ;)  Can figure out how we want to name the constants later.